hibernate-core/src/main/java/org/hibernate/type/descriptor/java/LocalDateTimeJavaDescriptor.java

```diff
@@ -98,7 +97,7 @@ public LocalDateTime fromString(String string) {
 
 		if ( Timestamp.class.isInstance( value ) ) {
 			final Timestamp ts = (Timestamp) value;
-			return LocalDateTime.ofInstant( ts.toInstant(), ZoneId.systemDefault() );
+			return ts.toLocalDateTime();
```

Comment:
Does the JDK bug apply here? Should there be a comment about it?

Reply:
The bug applies, but I didn't add a comment because the bug-free solution is also the most obvious solution... I'll add a comment just in case.